cime_config/cesm/allactive/testlist_allactive.xml

```diff
@@ -285,7 +285,7 @@
   </test>
   <test name="PFS" grid="f09_g16" compset="B1850" testmods="allactive/default">
     <machines>
-      <machine name="bluewaters" compiler="pgi " category="prebeta"/>
+      <machine name="bluewaters" compiler="pgi" category="prebeta"/>
```

Comment:
This and some other similar changes to the testlists are just removing whitespace from the end of compiler names, which I noticed when testing the new utility. I'm not sure if these changes will affect testing: it's _possible_ that some tests will start being run that weren't being run before.